continuity.api/src/main/java/org/continuity/api/entities/deserialization/CustomMapDeserializer.java

```diff
@@ -0,0 +1,46 @@
+package org.continuity.api.entities.deserialization;
+
+import java.io.IOException;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+public class CustomMapDeserializer extends StdDeserializer<Map<String, String>> {
```

Comment:
For me, this looks like the default Jackson deserializer. Do you really need it?

Reply:
Japp, this necessary, because I either want to provide, that the dedicated services of a modularized load test can be defined as : 
`services:
   -tag1
   -tag2`

or as 
`services:
   tag1 : hostname1
   tag2 : hostname2